circle.yml

```diff
@@ -0,0 +1,27 @@
+machine:
+  services:
+    - docker
+
+dependencies:
+  override:
+    - docker info
+
+    # create a version.json
+    - >
+        printf '{"commit":"%s","version":"%s","source":"https://github.com/%s/%s","build":"%s"}\n'
+        "$CIRCLE_SHA1"
+        "$CIRCLE_TAG"
+        "$CIRCLE_PROJECT_USERNAME"
+        "$CIRCLE_PROJECT_REPONAME"
+        "$CIRCLE_BUILD_URL"
+        > version.json
+
+    - docker build -t sops:build .
+
+deployment:
+  master:
+    branch: "master"
+    commands:
+      - docker login -e $DOCKER_EMAIL -u $DOCKER_USER -p $DOCKER_PASS
+      - "docker tag sops:build docker.io/mozilla/sops:latest"
```

Comment:
I think we should also tag and push with the git sha1 or the tag, because if we use latest rolling back deployments will be complicated.